unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -1866,6 +1866,29 @@ protected void updateAllAgentCollidersForVisibilityCheck(bool enableColliders)
             return Time.time;
         }
 
+        public bool objectIsWithinViewport(SimObjPhysics sop) {
```

Comment:
Can this be made private or protected? Does it need to be public?

Reply:
Made it protected.